addons/stock/static/src/scss/stock_location_hierarchy.scss

```diff
@@ -0,0 +1,100 @@
+$o-hierarchical-highlight-color: $o-brand-primary;
+$o-hierarchical-border-color: gray('500');
+$o-hierarchical-border-width: 1px;
+$o-hierarchical-border-radius: 4px;
+
+.o_stock_location_hierarchy {
+    .o_hierarchy_container {
+        border: 1px solid gray('300');
+        background-color: lighten(gray('200'), 2%);
+        border-radius: .5rem;
+        white-space: nowrap;
+        overflow-x: auto;
+    }
+    ul {
```

Comment:
@jat-odoo can you add a class to the ul on the template and make this selector more specific? ie ul.o_hierarchy_something


same for li

Reply:
Done